install.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+# curl https://github.com/srevinsaju/zap/raw/main/install.sh | sh
+
+
+# A Bash Script to auto fetch latest release from srevinsaju/zap releases
+
+echo
+echo "####################"
+echo
+echo  '~   ZAP Installer  ~'
+echo
+echo "####################"
+echo
+
+
+
+
+# Config
+REPO="srevinsaju/zap"
+
+
+# Clean
+if [ -f "./zap" ]; then
+	echo [~] Found old zap binary. Removing....
+	rm "./zap"
+fi
```

Comment:
Since we moved to `mktemp`, is this check still required?

Reply:
Really sorry i was just checking rn and it caught my eye too. I've removed it just now.

I haven't made many PR before, probably just a few. Thanks for being patient :)

Reply:
Haha, thats completely fine. I and you learnt a lot of new stuff, I liked the way you tried to solve the issue. You did really good work!

Reply:
Thanks 😄👍🏼 

and Happy Hacktober 😄